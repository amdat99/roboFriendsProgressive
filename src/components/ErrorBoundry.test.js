import { shallow } from 'enzyme';
import React from 'react';
import ErrorBoundry from './ErrorBoundry'

import {createSerializer} from 'enzyme-to-json';
import toJson from 'enzyme-to-json';

expect.addSnapshotSerializer(createSerializer({mode: 'deep'}));

	it('expect to render ErrorBoundry component', () => {
	// Method1
	expect(toJson(shallow(<ErrorBoundry />))).toMatchSnapshot();
	})


 
it('expect  componentDidCatch to return true', () =>{
	const wrapper = shallow(<ErrorBoundry/>)
	 expect(wrapper).toMatchSnapshot();
	 wrapper.instance().componentDidCatch()
	    expect((wrapper.state())).toEqual({hasError: true})
	})

