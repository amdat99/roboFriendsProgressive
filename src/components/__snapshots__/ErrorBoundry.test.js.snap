// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ecpect should component update to return false 1`] = `""`;

exports[`expect  componentDidCatch to return true 1`] = `""`;

exports[`expect to render ErrorBoundry component 1`] = `""`;
